### PR TITLE
SAM-2865: Student response Pop-up with wrong format

### PR DESCRIPTION
**Pull request by ramonchu2k**

### CODE REVIEW

File: samigo/samigo-app/src/webapp/jsf/evaluation/questionScore.jsp [hunk 1]
```diff
@@ -951,7 +951,7 @@ function hiddenLinkOnClick(){
       
     <h:panelGroup rendered="#{questionScores.selectedSARationaleView == '1' && questionScores.typeId == '5'}">
     <h:outputText value="#{description.answer}" escape="false"/>
-		<h:outputLink title="#{evaluationMessages.t_fullShortAnswer}"   value="#" onclick="javascript:window.open('fullShortAnswer.faces?idString=#{description.assessmentGradingId}','fullShortAnswer','width=600,height=600,scrollbars=yes, resizable=yes');" onkeypress="javascript:window.open('fullShortAnswer.faces?idString=#{description.assessmentGradingId}','fullShortAnswer','width=600,height=600,scrollbars=yes, resizable=yes');">
+		<h:outputLink title="#{evaluationMessages.t_fullShortAnswer}"   value="#" onclick="javascript:window.open('/portal/tool/#{requestScope['sakai.tool.placement.id']}/jsf/evaluation/fullShortAnswer.faces?idString=#{description.assessmentGradingId}','fullShortAnswer','width=600,height=600,scrollbars=yes, resizable=yes');" onkeypress="javascript:window.open('/portal/tool/#{requestScope['sakai.tool.placement.id']}/jsf/evaluation/fullShortAnswer.faces?idString=#{description.assessmentGradingId}','fullShortAnswer','width=600,height=600,scrollbars=yes, resizable=yes');">
```
Comment (bjones86) [inline samigo/samigo-app/src/webapp/jsf/evaluation/questionScore.jsp]: The `javascript:` prefix is only necessary when executing JavaScript inside an `href=""` attribute.

**Comment by bjones86** [comment]
Very minor suggestion, otherwise LGTM

**Comment by ramonchu2k** [comment]
Fixed, thanks for the suggestion.

File: samigo/samigo-app/src/webapp/jsf/evaluation/questionScore.jsp [hunk 2]
```diff
@@ -951,7 +951,7 @@ function hiddenLinkOnClick(){
       
     <h:panelGroup rendered="#{questionScores.selectedSARationaleView == '1' && questionScores.typeId == '5'}">
     <h:outputText value="#{description.answer}" escape="false"/>
-		<h:outputLink title="#{evaluationMessages.t_fullShortAnswer}"   value="#" onclick="javascript:window.open('fullShortAnswer.faces?idString=#{description.assessmentGradingId}','fullShortAnswer','width=600,height=600,scrollbars=yes, resizable=yes');" onkeypress="javascript:window.open('fullShortAnswer.faces?idString=#{description.assessmentGradingId}','fullShortAnswer','width=600,height=600,scrollbars=yes, resizable=yes');">
+		<h:outputLink title="#{evaluationMessages.t_fullShortAnswer}"   value="#" onclick="window.open('/portal/tool/#{requestScope['sakai.tool.placement.id']}/jsf/evaluation/fullShortAnswer.faces?idString=#{description.assessmentGradingId}','fullShortAnswer','width=600,height=600,scrollbars=yes, resizable=yes');" onkeypress="javascript:window.open('/portal/tool/#{requestScope['sakai.tool.placement.id']}/jsf/evaluation/fullShortAnswer.faces?idString=#{description.assessmentGradingId}','fullShortAnswer','width=600,height=600,scrollbars=yes, resizable=yes');">
```
Comment (bjones86) [inline samigo/samigo-app/src/webapp/jsf/evaluation/questionScore.jsp]: There's still one `javascript:` prefix in the `onkeypress` event handler, sorry!

**Comment by ramonchu2k** [comment]
Sorry, I didn't realize. Fixed.

**Comment by bjones86** [comment]
Thanks @ramonchu2k!

**Pull request merged**